### PR TITLE
Acreage image and text edits

### PR DESCRIPTION
**Pull request by jordansread**
placeholder before #116 is taken care of. I've let this branch veer off and included text edits. Whoops. I will clean it up, but wanted to let this hang so the area sankey code is available.

### CODE REVIEW

**Comment by jordansread** [comment]
Now is up to date w/ text edits

File: scripts/process/processFutureSuitability.R [hunk 1]
```diff
@@ -3,33 +3,54 @@ library(dplyr)
 
 processData.processFutureSuitability <- function(futureSuitability, outfile, ...){
 
+  area.factor = 0.000247105 # in acres
   fish.change <- futureSuitability %>% 
     rename(X1989.2014 = `1989-2014`, X2040.2064 = `2040-2064`, X2065.2089 = `2065-2089`) %>% 
     filter(!is.na(X1989.2014) & !is.na(X2040.2064) & !is.na(X2065.2089)) %>% 
     mutate(X1989.2014 = as.factor(X1989.2014), X2040.2064 = as.factor(X2040.2064),
-           X2065.2089 = as.factor(X2065.2089)) 
+           X2065.2089 = as.factor(X2065.2089), area = mda.lakes::getArea(WBIC)) %>% select(WBIC, area, everything())
```
Comment (jordansread) [inline scripts/process/processFutureSuitability.R:11]: This requires the older version of this package, which should be specified.

**Comment by jordansread** [comment]
This is now ready for review

File: layout/js/app.js [hunk 2]
```diff
@@ -3,4 +3,21 @@ document.firefoxSVGReload = function(id){
...
 };
+
+//Changes selected button colors
+$('.figToggle').each(function(){
+  $('.figToggle').on('click', function(){
+    $('.figToggle').removeClass('showing');
+    $(this).addClass('showing');
+    //toggles the figs based on which button is active
+    if($('#figArea').hasClass('showing')){
+      $('#futureSuitabilityFig').fadeOut(500);
```
Comment (thongsav-usgs) [inline layout/js/app.js]: To get this to not do the the funky toggle use the callback. EG:

```
$('#futureSuitabilityFig').fadeOut(500, function() {
    $('#futureSuitabilityFigArea').fadeIn(1000);
});
```

**Pull request merged**